src/rnn.jl

```diff
@@ -235,6 +240,9 @@ function rnnparam(r::RNN, w, layer::Integer, id::Integer, par::Integer; handle=c
             for l = 1:layer, i = 1:ids
                 if inputLayer(r,l) && i <= inputIds
                     len = (r.inputMode==0 ? XH : 0)
+                elseif l>2 && r.direction==1 && i <= div(ids, 2)
+                    # bidirectional weight
+                    len = 2HH
```

Comment:
Isn't this two separate weights, each HH in size, according to cudnn interface?

Reply:
```
julia> r,w = rnninit(5,5; numLayers=2, rnnType=:relu, bidirectional=true);

julia> for mat in rnnparams(r,w)
           println(size(mat))
       end
(5, 5)
(5, 5)
(5, 5)
(5, 5)
(10, 5)
(5, 5)
(10, 5)
(5, 5)
(5,)
(5,)
(5,)
(5,)
(5,)
(5,)
(5,)
(5,)
```